lua/lua-core/intro-to-lua/intro-code-lua.md

```diff
@@ -17,7 +17,9 @@ revisionQuestion:
 ---
 ## Content
 
-Like with every other language, let's write our first "hello world" program!
+Like with every other language, let's write our first "hello world" program to help you get used to coding!
+
+> If this confuses you, don't worry! We'll go much more in depth in future lessons!

```

Comment:
I think the one liner suffices here. 

Can we add the "if this confuses you" text as footnotes instead?

Reply:
Agreed on keeping things short and sweet.

I think it's helpful for @junoocha to also share why this was added (it's based on user's comments).

@HermenegildoMI Assuming the sufficient one-liner is "We'll go much more in depth in future lessons!", are you proposing to put the tiny sentence `"If this confuses you, don't worry!"` into a footnote? If so, not sure if that's worth having a footnote?

Reply:
I am going to stop myself in my tracks here. I actually am not aware of the user feedback. Please ignore my comments. 

I saw the previews generated and felt like the content was too long. Please ignore my comments. Think you have more context @nem035 to review 👍

Reply:
The comments I've seen are asking a lot about the code itself, the semantics, and the purpose, along with some confusion mixed in with the comments. 

As I simply wanted this insight to expose Lua code for the first time to users, I wanted to emphasize that point, as the concepts used in the code excerpt will properly be explained in later insights. Nevertheless, I agree that it's also important to ensure that the content is concise and the changes Hermen made were perfectly fine!

When I make future changes, I'll provide more context on why I made some changes. I appreciate the small edits!